.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,12 +1,7 @@
 ---
 name: Question
-about: Please ask questions in the ReSwitched discord, instead of making issues.
-
----
-
-We would like to use GitHub to keep track of problems/feature requests. 
-
-If you have a question, please join the ReSwitched discord for help.
-
-- Discord link: https://discordapp.com/invite/DThbZ7z
-
+about: Please ask questions in the ReSwitched discord, instead of making issues. 
+--- 
+We would like to use GitHub to keep track of problems/feature requests.  
+If you have a question, please join the ReSwitched discord for help. 
+- Discord link: https://discordapp.com/invite/DThbZ7z 
```

Comment:
Perhaps we should put a badge like this on the README: [![Chat on Discord](https://img.shields.io/badge/chat-Discord-brightgreen.svg)](https://discordapp.com/invite/ZdqEhed)

Reply:
That would be a much better solution rather than have the issue.

Reply:
Added a the provided discord badge to README.md